test/gentlemanjerry.bats

```diff
@@ -149,3 +178,28 @@ teardown() {
   run timeout 10 java -cp /tmp/test SslTest api.logentries.com 25414
   [ "$status" -eq 0 ]
 }
+
+@test "Gentleman Jerry can connect to a TLSv1.2-only Endpoint" {
+  wait_for_tls12_server
+  run timeout 5 openssl s_client -CAfile /tmp/certs/jerry.crt -connect localhost:4433
+  [[ "$output" =~ "Verify return code: 0 (ok)" ]]
+
+  # Import cert into test truststore
+  keytool -importcert -file /tmp/certs/jerry.crt -keystore /tmp/certs/jerry.jks -storepass changeit -noprompt
+
+  # Set up JRuby and its gems
+  export PATH="/logstash-$LOGSTASH_VERSION/vendor/jruby/bin:$PATH"
+  export GEM_PATH="/logstash-$LOGSTASH_VERSION/vendor/bundle/jruby/1.9"
+
+  run jruby /tmp/test/manticore_test.rb https://localhost:4433 /tmp/certs/jerry.jks
+  [ "$status" -eq 0 ]
+}
+
+@test "Gentleman Jerry can connect to a TLSv1.2-only Endpoint (variant)" {
+  # Set up JRuby and its gems
+  export PATH="/logstash-$LOGSTASH_VERSION/vendor/jruby/bin:$PATH"
+  export GEM_PATH="/logstash-$LOGSTASH_VERSION/vendor/bundle/jruby/1.9"
+
+  run jruby /tmp/test/manticore_test.rb https://tlsv12-elb.aptible-test-grumpycat.com
+  [ "$status" -eq 0 ]
```

Comment:
bats runs tests with `-o errexit` (which is why your `[ "$status" -eq 0 ]` check works to begin with), so I would recommend against using `run ...` if you're only going to check the status.

In other words, replace:

```
run jruby /tmp/test/manticore_test.rb https://tlsv12-elb.aptible-test-grumpycat.com
[ "$status" -eq 0 ]
```

With:

```
jruby /tmp/test/manticore_test.rb https://tlsv12-elb.aptible-test-grumpycat.com
```